pkg/metrics/metrics.go

```diff
@@ -0,0 +1,86 @@
+package metrics
+
+import (
+	"strconv"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var prometheusMetrics = false
+
+const (
+	lassoSubsystem      = "lasso_controller"
+	controllerNameLabel = "controllerName"
```

Comment:
I think the common case is snake for label names, any particular reason for the camel case here?

Reply:
Nope, will change.

Reply:
@ryansann updated this label and the ones below it to use snake case.